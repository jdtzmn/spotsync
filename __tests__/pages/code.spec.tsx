import * as React from 'react'
import { shallow } from 'enzyme'
import Code from 'pages/code'

describe('Code', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<Code />)
  })

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
