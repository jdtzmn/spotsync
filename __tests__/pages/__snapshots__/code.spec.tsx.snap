// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Code should render correctly 1`] = `
<main>
  <div>
    <div>
      <LinkRoutes
        route="/find"
      >
        <span>
          ‚Üê Choose a different option
        </span>
      </LinkRoutes>
      <h1>
        Enter Code
      </h1>
    </div>
    <div>
      <div>
        <Component
          autoFocus={true}
          key="0"
          onChange={[Function]}
          onKeyDown={[Function]}
          options={
            Object {
              "blocks": Array [
                1,
              ],
              "numericOnly": true,
            }
          }
          pattern="[0-9]*"
          placeholder="0"
          type="number"
        />
        <Component
          autoFocus={false}
          key="1"
          onChange={[Function]}
          onKeyDown={[Function]}
          options={
            Object {
              "blocks": Array [
                1,
              ],
              "numericOnly": true,
            }
          }
          pattern="[0-9]*"
          placeholder="0"
          type="number"
        />
        <Component
          autoFocus={false}
          key="2"
          onChange={[Function]}
          onKeyDown={[Function]}
          options={
            Object {
              "blocks": Array [
                1,
              ],
              "numericOnly": true,
            }
          }
          pattern="[0-9]*"
          placeholder="0"
          type="number"
        />
        <Component
          autoFocus={false}
          key="3"
          onChange={[Function]}
          onKeyDown={[Function]}
          options={
            Object {
              "blocks": Array [
                1,
              ],
              "numericOnly": true,
            }
          }
          pattern="[0-9]*"
          placeholder="0"
          type="number"
        />
        <Component
          autoFocus={false}
          key="4"
          onChange={[Function]}
          onKeyDown={[Function]}
          options={
            Object {
              "blocks": Array [
                1,
              ],
              "numericOnly": true,
            }
          }
          pattern="[0-9]*"
          placeholder="0"
          type="number"
        />
        <Component
          autoFocus={false}
          key="5"
          onChange={[Function]}
          onKeyDown={[Function]}
          options={
            Object {
              "blocks": Array [
                1,
              ],
              "numericOnly": true,
            }
          }
          pattern="[0-9]*"
          placeholder="0"
          type="number"
        />
      </div>
    </div>
  </div>
</main>
`;
